pkg/project/api/v1beta1/types.go

```diff
@@ -31,7 +31,7 @@ type ProjectStatus struct {
 type Project struct {
 	kapi.TypeMeta   `json:",inline"`
 	kapi.ObjectMeta `json:"metadata,omitempty"`
-	DisplayName     string `json:"displayName,omitempty"`
+	//DisplayName, Description are Annotations (in kapi.ObjectMeta struct)
```

Comment:
v1beta1 types.go file does not change.

Keep DisplayName as it was in v1beta1/types.go